dae/dae/variants/family.py

```diff
@@ -60,6 +60,36 @@ def has_attr(self, item):
     def get_attr(self, item):
         return str(self.atts.get(item))
 
```

Comment:
consider moving this code to Study/StudyWrapper

Reply:
This is moved to Study